go/chat/types/types.go

```diff
@@ -38,6 +39,8 @@ const (
 	MapsDomain = "keybasemaps"
 )
 
+var EmojiPattern = regexp.MustCompile(`(?::)([^:\s]+)(?::)`)
```

Comment:
This should probably go in `globals`, I usually only put stuff in type that show up in the interfaces. `DefaultTeamTopic` is like this and is in globals.

Reply:
done